app/assets/stylesheets/home.scss

```diff
@@ -40,113 +57,111 @@ h3 {
   background-color: #00796B;
 }
 
-#team {
-  background: image_url("success_team.png");
-  background-size: cover;
-  height: 150px;
-  width: 150px;
+li.team {
+  text-align: center;
+  padding-right: 0;
+  padding-left: 25px;
+  width: 250px;
 }
 
-.site-footer {
-  background-color: #00796B;
-  color: white;
+#team {
+  padding-right: 0;
 }
 
-* {
-	margin: 0;
-}
-html, body {
-	height: 100%;
+li.search {
+  text-align: center;
+  margin-top: 20px;
+  margin-right: 27%;
+  float: right;
+  max-width: 300px;
+  padding: 0 20px 0 0;
 }
-.page-wrap {
-  min-height: 100%;
-  /* equal to footer height */
-	margin-bottom: -99px;
+
+form #q {
+  margin-bottom: 10px;
 }
-.page-wrap:after {
-  content: "";
-  display: block;
+
+form .tiny {
+  margin-bottom: 0;
 }
-.site-footer, .page-wrap:after {
-  /* .push must be the same height as footer */
-  height: 65px;
-  margin-top: 10px;
+
+.site-footer {
+  background-color: #00796B;
+  color: white;
+  height: 70px;
 }
 
-#featured {
-  background-color: #009688;
-  height: 400px;
-  width: 100%;
-  text-align: center;
+.site-footer hr {
+  margin-bottom: 10px;
 }
 
-.large-3 {
-  height: 220px;
-  margin: 8px 0;
+.site-footer a {
+  color: #E0F2F1;
   transition: all 0.15s ease-in-out;
 }
 
-.large-3:hover {
-  opacity: 0.8;
+.site-footer a:hover {
+  opacity: 0.54;
   transform: scale(1.08);
   z-index: +1;
 }
 
-.large-3 .mini-1 {
-  background-color: #e74c3c;
-  width: 100%;
-  padding: 67.5px 0 67.5px 0;
+.featured-content a {
+  color: white;
+  transition: all 0.15s ease-in-out;
 }
 
-.large-3 .mini-2 {
-  background-color: #3498db;
-  width: 100%;
-  padding: 67.5px 0 67.5px 0;
+.featured-content a:hover {
+  opacity: 0.54;
+  transform: scale(1.08);
+  z-index: +1;
 }
 
-.large-3 .mini-3 {
-  background-color: #9b59b6;
-  width: 100%;
-  padding: 67.5px 0 67.5px 0;
+* {
+	margin: 0;
 }
-
-.large-3 .mini-4 {
-  background-color: #e67e22;
-  width: 100%;
-  padding: 67.5px 0 67.5px 0;
+html, body {
+	height: 100%;
+}
+.page-wrap {
+  min-height: 100%;
+  /* equal to footer height */
+  padding: 0;
+}
+.page-wrap:after {
+  content: "";
+  display: block;
 }
 
-#soccer {
-  background: image_url("soccer_ball.png");
+#featured {
+  background: image_url("blurred_wallpaper.jpg") no-repeat center center;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size, color, height, text-align, width